src/Controls/samples/Controls.Sample.UITests/Issues/Issue19329.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 19329, "Pointer gestures should work with relative positions correctly", PlatformAffected.iOS)]
+	public partial class Issue19329 : ContentPage
+	{
+		public Issue19329()
+		{
+			InitializeComponent();
+		}
+
+		private void OnTapped(object sender, TappedEventArgs e)
+		{
+			// Report that the callback was called.
+			layout.Children.Add(new Label { Text = "TapAccepted", AutomationId = "TapAccepted" });
+
+			Point? position = e.GetPosition(relativeTo: UpperBox);
+
+			string result;
+			string automationId;
+
+			if (position is null)
+			{
+				result = "Error: position is null";
+				automationId = "Failure";
+			}
+			else if (position.Value.X > 40 && position.Value.Y > 20)
```

Comment:
I think the nature of the test might have changed some as we went through controls.

When I set this to a label it basically just resulted in two labels that were lined up vertically which meant that this test was passing even without your change. 

The current issue is that GetPosition is always just returning the position relative to the view the gesture is attached to. 

I modified the test so that the two labels were on opposite ends, and then it verifies that the position returned is > bounds of the current view. 

![image](https://github.com/dotnet/maui/assets/5375137/e848ee41-6041-40a6-9325-6b2af34dc4b0)

The upper bounds is a good idea though. 

I've modified the test to make sure it's inside the expected range

![image](https://github.com/dotnet/maui/assets/5375137/4be77dcf-0568-4ed2-a81d-af0e855ac909)